### DISCUSSION TITLE
Azure SQL workspace service template

### DISCUSSION THREAD

**Discussion by jonnyry** [microsoft/AzureTRE#3936] [Ideas]
Has anyone built or is anyone considering building an Azure SQL workspace service template?

Having access to a T-SQL instance within a workspace is something that keeps coming up locally... just wondering if its something that has been considered?

**Comment by marrobi** [comment]
We've had users connect to MS SSQL instances outside of workspaces, as part of a data platform, but nobody actually need one inside a workspace, has all been MySQL or Postgres. Shouldn't be too tricky to create one and would welcome a PR.

**Comment by jonnyry** [comment]
OK thanks... I will provide a PR if we do indeed go ahead with an AzureSQL workspace service.

**Comment by jonnyry** [comment]
Create as feature request #3969 and sent PR #3970.
